packages/dx-react-grid-material-ui/src/templates/table-select-row.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TableRow } from 'material-ui';
+
+export const TableSelectRow = ({
+  tableRow,
+  children,
+  changeSelected,
+  selectByRowClick,
+  ...restProps
+}) => (
+  <TableRow
+    selected={tableRow.selected}
```

Comment:
Seems like selected prop should on the root options layer rather than inside tableRow.